tests/psa_helpers.function

```diff
@@ -0,0 +1,39 @@
+/*
+ * Helper functions for tests that use the PSA API.
+ */
+/*  Copyright (C) 2019, ARM Limited, All Rights Reserved
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  This file is part of mbed TLS (https://tls.mbed.org)
+ */
+
+#if defined(MBEDTLS_PSA_CRYPTO_SPM)
+#include "spm/psa_defs.h"
+#endif
+#include <psa/crypto.h>
+
+/** Evaluate an expression and fail the test case if it returns an error.
+ *
+ * \param expr      The expression to evaluate. This is typically a call
+ *                  to a \c psa_xxx function that returns a value of type
+ *                  #psa_status_t.
+ */
+#define PSA_ASSERT( expr ) TEST_EQUAL( ( expr ), PSA_SUCCESS )
+
+/*
+ * Local Variables:
```

Comment:
What is this comment for?

Reply:
[Emacs](https://www.gnu.org/software/emacs/manual/html_node/emacs/Specifying-File-Variables.html) (and if someone wants to add the Vi(m) equivalent, be my guest).

Reply:
I have this for the other `.function` files:
```
(add-to-list 'auto-mode-alist '("/suites/[^/]+\\.function\\'" . c-mode))
```
I could do the same for `tests/*.function` but it's very generic, `suites` is a lot less common.

Ideally I'd prefer to do this from a `.dir-locals.el` file, but unfortunately you can't set the mode of a specific file in `.dir-locals.el`.

Reply:
I don't intend to object to this personally, I just think it is not in line with how the library has been developed so far, hence I pointed it out.